test/tools/Dockerfile

```diff
@@ -46,3 +46,8 @@ RUN aws --version
 # Install eksctl
 RUN curl --silent --location "https://github.com/weaveworks/eksctl/releases/download/v0.105.0/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
 RUN mv /tmp/eksctl /usr/local/bin
+
+# Install toolchain
+RUN git clone https://github.com/aws/karpenter.git /karpenter
+WORKDIR /karpenter
+RUN GOPRIVATE=* go mod tidy
```

Comment:
You shouldn't set `GOPRIVATE=*` since this causes go to skip checksum verifications... Do you just want the proxy off or is there more you're trying to do here?